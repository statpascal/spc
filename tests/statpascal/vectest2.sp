program vectest2;

const
    n = 10;

var
    v: vector of string;
    s: string;
    i: 1..n;

begin
    for i := 1 to n do
	begin
	    str (i, s);
	    v := combine (v, 'This is string ' + s)
	end;
    for i := 1 to size (v) do
	writeln (v [i])
end.

