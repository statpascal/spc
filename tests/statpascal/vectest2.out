This is string 1
This is string 2
This is string 3
This is string 4
This is string 5
This is string 6
This is string 7
This is string 8
This is string 9
This is string 10
